### ISSUE TITLE
Error when running my own TFLite model

### ISSUE THREAD

**Issue by chicolinux** [oscarlab/graphene#2637]
## Description of the problem
I am getting the following error when running my own TFLite model:
tensorflow/contrib/lite/kernels/reshape.cc:68 num_input_elements != num_output_elements (3200 != 0)
Failed to allocate tensors!make: *** [Makefile:88: run-graphene] Error 139

This is related to the Tensorflow Lite EXAMPLE that comes with graphene!

I guess because the label_image application is expecting an image of a different size than the input to the ML model of (32, 32, 3), input_shape=  (32, 32, 3)

Checking the code for the label_image application I see the following:
  int image_width = 224;
  int image_height = 224;
  int image_channels = 3;

Should I increase the size of the image I am using to train my model?

I attached the TFLite model file!

## Steps to reproduce

make SGX=1 run-graphene
[model_originalv2.tflite.zip](https://github.com/oscarlab/graphene/files/6959426/model_originalv2.tflite.zip)

## Actual results
tensorflow/contrib/lite/kernels/reshape.cc:68 num_input_elements != num_output_elements (3200 != 0)
Failed to allocate tensors!make: *** [Makefile:88: run-graphene] Error 139

## Additional information
This is my model:

    #Standard neural network training procedure.     params: [64, 64, 128, 128, 256, 256]

    print("-------------------------------------------------------\n")
    model = models.Sequential()

    print(data.train_data.shape) # For CIFAR-10 (45000,32,32,3)
    
    model.add(Conv2D(params[0], (3, 3), input_shape=  (32, 32, 3) ))
    model.add(Activation('relu'))
    model.add(Conv2D(params[1], (3, 3)))
    model.add(Activation('relu'))
    model.add(MaxPooling2D(pool_size=(2, 2)))

    model.add(Conv2D(params[2], (3, 3)))
    model.add(Activation('relu'))
    model.add(Conv2D(params[3], (3, 3)))
    model.add(Activation('relu'))
    model.add(MaxPooling2D(pool_size=(2, 2)))

    model.add(Flatten())
    model.add(Dense(params[4]))
    model.add(Activation('relu'))
    model.add(Dropout(0.5))
    model.add(Dense(params[5]))
    model.add(Activation('relu'))
    model.add(Dense(10))

**Comment by dimakuv** [comment]
Unfortunately, this doesn't sound like a Graphene issue but rather a mismatch between the `label_image` example application and your own model.

We (Graphene developers) are not professionals in TensorFlow Lite and in AI/ML workloads in general, so I don't think anyone here can help with your issue. You should first try to make the original (native) application work for you, and then try it under Graphene.

> Should I increase the size of the image I am using to train my model?

Probably. Try it out. Unfortunately, that's not a question relating to Graphene, so can't help much.

(By the way, if your original application actually works, and only the Graphene runs do not work, then maybe you need to increase `sgx.enclave_size`?)

**Comment by mkow** [comment]
@dimakuv: But I think he says that this happens with the model included in our examples?

**Comment by chicolinux** [comment]
No, @dimakuv is right! I am using my own TFLite model!! Thank you guys! I guess you can close this issue! By the way, It will nice to update the documentation at state that the label_image demo app, expects images >= 224x224x3

**Comment by mkow** [comment]
Ah. I was confused by "I am getting the following error when running my own TFLite model" vs "This is related to the Tensorflow Lite EXAMPLE that comes with graphene!" :)
